### PR TITLE
Fix typos in preface

### PR DESCRIPTION
**Pull request by rjmill**
Thanks for writing this book! I'm really enjoying it so far.

I'm fixing any typos and grammatical errors I catch as I work my way through the book. These are my fixes for the preface.

I kept each fix in a separate commit to make it easier to review, but I can squash them all together if you'd prefer to keep the commit history less noisy.

### CODE REVIEW

**Comment by rjmill** [comment]
And please let me know if there are any places where you disagree with how I "fixed" something. I tried to keep my changes as close to your words as possible, but there were some places where I might have deviated from your intentions.

File: book_src/Operating System From 0 to 1.lyx [hunk 1]
```diff
@@ -502,7 +502,7 @@ There are many books on this topic and even courses out there already, made
  book, and probably the students are assumed to obtain enough hardware knowledge
  beforehand from other courses.
  Or, the students only need to work within existing assignment framework:
- the only thing they need to do is filling the empty function body, run
+ the only thing they need to do is fill the empty function body, run
```
Comment (tuhdo) [inline book_src/Operating System From 0 to 1.lyx]: Isn't `fill` a verb already?

**Comment by tuhdo** [comment]
Thank you very much!

File: book_src/Operating System From 0 to 1.lyx (hunk 1 above)
Comment (rjmill) [inline book_src/Operating System From 0 to 1.lyx]: I wish I could give you a more technical answer, but I'm honestly not sure of the exact reasons why "fill" is correct here. It's correct for the same reasons "run" and "complete" are correct in the same sentence though.

**Pull request merged**